### ISSUE TITLE
how to pretrained TSN-model?

### ISSUE THREAD

**Issue by longweiwei** [open-mmlab/mmaction#55]
hi @zhaoyue-zephyrus 
I find training loss  dropped very quickly on TSN-rgb-model when I use the pre-trainind weights to initializate. while I train the model from scratch,the training loss dropped very slow . That's amazing! So I really want to know how you pre-train the model(such as tsn-rgb-bninception,tsn-flow-bninception,i3d,...).I hope you can tell me! 
thanks you!

**Comment by longweiwei** [comment]
and when i use the pre-trainin model ,there are many unmatch keys:

missing keys in source state_dict: inception_5b_3x3_reduce_bn.num_batches_tracked, inception_4c_1x1_bn.num_batches_tracked, inception_5b_3x3_bn.num_batches_tracked, inception_3c_double_3x3_1_bn.num_batches_tracked, inception_4a_3x3_bn.num_batches_tracked, inception_3b_1x1_bn.num_batches_tracked, inception_3a_pool_proj_bn.num_batches_tracked, inception_3a_1x1_bn.num_batches_tracked, inception_5a_1x1_bn.num_batches_tracked, inception_4d_double_3x3_reduce_bn.num_batches_tracked, inception_3c_double_3x3_2_bn.num_batches_tracked, conv1_7x7_s2_bn.num_batches_tracked, inception_5a_3x3_reduce_bn.num_batches_tracked, inception_4c_3x3_reduce_bn.num_batches_tracked, inception_4b_double_3x3_reduce_bn.num_batches_tracked, conv2_3x3_reduce_bn.num_batches_tracked, inception_3a_double_3x3_2_bn.num_batches_tracked, inception_4c_double_3x3_1_bn.num_batches_tracked, inception_4b_double_3x3_1_bn.num_batches_tracked, inception_3b_double_3x3_1_bn.num_batches_tracked, inception_4b_1x1_bn.num_batches_tracked, inception_5b_double_3x3_2_bn.num_batches_tracked, inception_3c_3x3_bn.num_batches_tracked, inception_4d_double_3x3_1_bn.num_batches_tracked, inception_4c_pool_proj_bn.num_batches_tracked, inception_4e_double_3x3_2_bn.num_batches_tracked, inception_4e_3x3_bn.num_batches_tracked, inception_5b_double_3x3_1_bn.num_batches_tracked, inception_3b_double_3x3_2_bn.num_batches_tracked, inception_5a_pool_proj_bn.num_batches_tracked, inception_5a_3x3_bn.num_batches_tracked, inception_3a_3x3_bn.num_batches_tracked, inception_3b_3x3_reduce_bn.num_batches_tracked, inception_3b_double_3x3_reduce_bn.num_batches_tracked, inception_4a_3x3_reduce_bn.num_batches_tracked, inception_5b_double_3x3_reduce_bn.num_batches_tracked, inception_4d_pool_proj_bn.num_batches_tracked, inception_5b_pool_proj_bn.num_batches_tracked, inception_3c_3x3_reduce_bn.num_batches_tracked, inception_4c_double_3x3_2_bn.num_batches_tracked, inception_5b_1x1_bn.num_batches_tracked, inception_4d_3x3_bn.num_batches_tracked, inception_4a_double_3x3_1_bn.num_batches_tracked, inception_3b_pool_proj_bn.num_batches_tracked, inception_4e_3x3_reduce_bn.num_batches_tracked, inception_3a_double_3x3_1_bn.num_batches_tracked, inception_4a_1x1_bn.num_batches_tracked, inception_3a_double_3x3_reduce_bn.num_batches_tracked, inception_3c_double_3x3_reduce_bn.num_batches_tracked, inception_3b_3x3_bn.num_batches_tracked, conv2_3x3_bn.num_batches_tracked, inception_4e_double_3x3_1_bn.num_batches_tracked, inception_4c_3x3_bn.num_batches_tracked, inception_5a_double_3x3_reduce_bn.num_batches_tracked, inception_4c_double_3x3_reduce_bn.num_batches_tracked, inception_4d_double_3x3_2_bn.num_batches_tracked, inception_4a_double_3x3_2_bn.num_batches_tracked, inception_5a_double_3x3_1_bn.num_batches_tracked, inception_4b_3x3_reduce_bn.num_batches_tracked, inception_4d_3x3_reduce_bn.num_batches_tracked, inception_4b_3x3_bn.num_batches_tracked, inception_4e_double_3x3_reduce_bn.num_batches_tracked, inception_3a_3x3_reduce_bn.num_batches_tracked, inception_4b_double_3x3_2_bn.num_batches_tracked, inception_4a_double_3x3_reduce_bn.num_batches_tracked, inception_4b_pool_proj_bn.num_batches_tracked, inception_4d_1x1_bn.num_batches_tracked, inception_5a_double_3x3_2_bn.num_batches_tracked, inception_4a_pool_proj_bn.num_batches_tracked

**Comment by zhaoyue-zephyrus** [comment]
@longweiwei Sorry about the late reply.

1. The pre-trained (2D) rgb model is simply a CNN trained on the ImageNet. The trick of `inflation` to adapt the original 2D rgb model to the flow model and to I3D can be found in the paper of [TSN](https://arxiv.org/abs/1705.02953) and [I3D](https://arxiv.org/abs/1705.07750) respectively.

2. You can simply ignore the warning since these params have no effect.

**Comment by longweiwei** [comment]
Got  it! thanks you@zhaoyue-zephyrus

**Comment by Anirudh257** [comment]
@longweiwei @zhaoyue-zephyrus which exact checkpoint did you use to extract the **RGB BN-inception features** for **EK-100**? I am unable to find the **bn_inception** model from here: https://github.com/open-mmlab/mmaction2/tree/0.x/configs/recognition/tsn
